### ISSUE TITLE
Suggested change to wording re bookmarks and object position.

### ISSUE THREAD

**Issue by jeffreyweir** [MicrosoftDocs/powerbi-docs#5019]
The bullet point _If you move a visual after you create a bookmark, when you select the bookmark, the visual appears where you moved it_ is hard to parse, and not technically correct. 

Visuals don't "appear where you moved it". 

Rather, bookmarks don't store object position, and so don't _change_ where you moved it.

I would say "Bookmarks have no impact on the position of objects. If you subsequently change the location of bookmarked objects, triggering the bookmark will not return those objects to their previous position."

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 6e579557-27e4-8a24-69b7-55189b0f4966
* Version Independent ID: 0c97fa47-9695-aaaf-efcd-63b35362310e
* Content: [Create report bookmarks in Power BI to share insights and build stories - Power BI](https://learn.microsoft.com/en-us/power-bi/create-reports/desktop-bookmarks?tabs=powerbi-desktop)
* Content Source: [powerbi-docs/create-reports/desktop-bookmarks.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/main/powerbi-docs/create-reports/desktop-bookmarks.md)
* Service: **powerbi**
* Sub-service: **pbi-reports-dashboards**
* GitHub Login: @mihart
* Microsoft Alias: **maggies**

**Comment by bishalgoswami** [comment]
Thanks for your comment, @jeffreyweir. This issue has been assigned to the doc author for review.

**Comment by mihart** [comment]
Hi @jeffreyweir , 
Thank you for your suggestion. We really appreciate it. We've made the update and set the article to republish. 
All the best,
The DOCS team
